cover/npm-cli.txt

```diff
@@ -0,0 +1,51 @@
+access
```

Comment:
@JelaniThompson please remove this file. The Keywords.txt is the only file you need 👍